### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by VivekRaaZ**
You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

Colors.✅

Typography: font face, size, and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design). 👌
Layout: composition and space between elements. 👍
The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.✅

You can use a CSS framework (for example Bootstrap) for styling, if you want to.✅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,180 @@
...
+  <h3><b>Microverse README Template</b></h3>
+  
+</div>
+# 📗 Table of Contents
+
+- [📖 About the project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```
Comment (mwanawabangona) [inline README.md:14]: - [ ] `Required`
- [ ] Amazing job so far 👏 👏 
- [ ] however, I noticed the deployed link is taking me to another project.
- [ ] Please see the screenshot below and make amends 🥂 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+              science technology, humanities, and the arts.
+            </p>
+          </div>
+        </div>
+      </div>
+        <button class="more">
+          MORE <img src="image/arrow_down 1.png" alt="" />
+        </button>
+      </aside>
+    </section>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Great Job here as well but😞 
- [ ] This section `Featured speakers` should be created dynamically in JavaScript.
- [ ] You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
- [ ] Please make amends. 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please Double check the requirements and see to it you have met them,
- before your next `PR`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @VivekRaaZ 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**